notify/src/inotify.rs

```diff
@@ -471,7 +474,8 @@ impl EventLoop {
                         for (w, p) in &self.paths {
                             if p.starts_with(&path) {
                                 inotify
-                                    .rm_watch(w.clone())
+                                    .watches()
```

Comment:
The watches() call seems to perform an Arc clone and creating some sort of Struct containing a [FdGuard](https://github.com/hannobraun/inotify-rs/blob/39b64297a6d3dd2706652c0e19d061287a4e04be/src/fd_guard.rs#L22). It might be in our interest to use one watches() call for at least the whole loop.

Reply:
Thanks, will look into this.

Reply:
Done in b70b72b79f9ad273a2f36a3d435a97f069b360ca